specification/source/cdl.rst

```diff
@@ -91,14 +91,22 @@ if used in a controller.
 ]
 
 Each of these data types, including the elementary building blocks,
-can be a single instance or one-dimensional array.
+can be a single instance, one-dimensional array or two-dimensional array (matrix).
 Array indices shall be of type ``Integer`` only.
 The first element of an array has index ``1``.
 An array of size ``0`` is an empty array.
-See the Modelica 3.3 specification Chapter 10 for array notation.
+
+Arrays may be constructed with the general array constructor,
```

Comment:
Not clear what you mean by "the general array constructor". Please rewrite.

Reply:
Sure. It is the wording from the Modelica specification.